ivy_tests/test_ivy/test_frontends/test_jax/test_numpy/test_mathematical_functions.py

```diff
@@ -1062,6 +1062,41 @@ def test_jax_ediff1d(
     )
 
 
+# einsum_path
+@handle_frontend_test(
+    fn_tree="jax.numpy.einsum_path",
+    eq_n_op_n_shp=helpers.einsum_helper(),
+    dtype=helpers.get_dtypes("numeric", full=False),
+    test_with_out=st.just(False),
+)
```

Comment:
Hi @rohitkg83, thanks for the PR. You should also generate values to test the `optimize` argument.

Reply:
Hey @AnnaTz - Thanks for reviewing this. I have now added optimize parameter to the tests. I haven't added boolean values to the samples as it seems that Jax einsum_path function currently fails when True or False is passed as optimize values. Jax einsum_path function calls opt_einsum.contract_path function and it seems that there is an open [bug](https://github.com/dgasmith/opt_einsum/issues/219) on their repository for boolean values. Hope this makes sense.

Reply:
Please add a comment above the function's test, explaining the issue and including the link.

Reply:
Done.